images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,180 @@
+
+function openvpn_config {
+	# Fectch UUID and Key of the default VPN only if they
+	# are not already set. The user may override the UUID and Key
+	# by setting them in the environment variables to use deploy
+	# a different VPN server.
+	if [ -z "$UUID" ]; then
+		export UUID=$(get_redis_value "openwisp_default_vpn_uuid")
+		export KEY=$(get_redis_value "openwisp_default_vpn_key")
+		export CA_UUID=$(get_redis_value "openwisp_default_vpn_ca_uuid")
+	fi
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Use 'foo()' instead of 'function foo'.

(SC2113)